web/themes/custom/votegov/src/sass/uswds-overrides/usa-language-selector.scss

```diff
@@ -31,8 +31,9 @@
 .usa-language__submenu {
   @include u-padding(2.5);
   @include u-width('auto');
-  @include u-radius('sm');
   @include u-text('no-wrap');
+  background-color: $base-dark;
+  border-radius: 2px 0 2px 2px;
```

Comment:
i used the native css property so that way we were not setting them all in one pace and then overriding one side